microsoft.ui.xaml.controls/navigationview_isbackbuttonvisible.md

```diff
@@ -11,7 +11,7 @@ public NavigationViewBackButtonVisible IsBackButtonVisible { get;  set; }
 
 ## -description
 
-Gets or sets a value that indicates whether the back button is enabled or disabled.
+Gets or sets a value that indicates whether the back button is visible or not. The value "Auto" indicates that the NavigationView determines whether the back button is visible or not depending on the display mode of the NavigationView.

```

Comment:
```suggestion
Gets or sets a value that indicates whether the back button is visible or not. Default value is "Auto", which indicates that the NavigationView determines whether the back button is visible or not depending on the display mode of the NavigationView.

```

Reply:
Good idea to note that auto is the default here.